include/Gaffer/ScriptNode.h

```diff
@@ -248,6 +248,12 @@ class GAFFER_API ScriptNode : public Node
 		/// > expression - always use `context.getFrame()` instead.
 		FloatPlug *framePlug();
 		const FloatPlug *framePlug() const;
+		/// The ScriptNode defines the valid frame range using two numeric plugs.
+		/// These drive the frameStart and frameEnd variables in the context.
```

Comment:
Needs updating to frameRange:start and frameRange:end - once you've done that could you squash things down and merge?

Reply:
Sure. Do you want the first commit squashed as well then?

Reply:
Sorry, when I saw multiple "change addressed in ..." comments I assumed there were multiple fixup commits rather than a new push, which is why my wording was probably confusing. I mean squash the upcoming fix into 8143029, and keep c5c9941 separate...

Reply:
Fixed in 8384f6c